src/hotspot/share/opto/superword.cpp

```diff
@@ -3807,3 +3036,412 @@ bool SuperWord::same_origin_idx(Node* a, Node* b) const {
 bool SuperWord::same_generation(Node* a, Node* b) const {
   return a != nullptr && b != nullptr && _clone_map.same_gen(a->_idx, b->_idx);
 }
+
+bool SuperWord::vtransform() const {
```

Comment:
Note: most similar code used to be `SuperWord::schedule()`. Call to `graph.apply()` is roughly what `SuperWord::output()` used to be.